### ISSUE TITLE
Define image resource things as "critical sub-resources"

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/manifest#910]
See: 
https://html.spec.whatwg.org/multipage/infrastructure.html#critical-subresources

Applies to shortcuts and icons.

**Comment by aarongustafson** [comment]
Make a note about these being critical & needing to be downloaded during processing.

**Comment by marcoscaceres** [comment]
We need to integrate into the processing algorithm, after processing, they image resources should be downloaded.

**Comment by aarongustafson** [comment]
We talked about using CSS as an example… progress with this has been slow over there: https://github.com/w3c/csswg-drafts/issues/1088

**Comment by aarongustafson** [comment]
@marcoscaceres Per https://github.com/whatwg/html/pull/5581#issuecomment-657622130, it seems like we may not need to do this task right now, no?

**Comment by marcoscaceres** [comment]
Correct, but we do need to do it when processing happens so definitely as part of CR... but happy to wait until scope and start URL are all sorted.

**Comment by marcoscaceres** [comment]
This is both applicable and not... firstly, upon reflection, we should not assume there is a client at all because the way this is commonly implemented is that the manifest structure is handed off to the OS to deal with... once it enters the OS, it no longer has a (web) client... it's potentially just using OS-level mechanisms to load the images. 

From a security perspective, the only assurances that we give is that it's some kind of image format (which yes, it may attempt to attack the OS) - but generally the image formats are considered safe, and CSP doesn't apply anymore as there is no document object or environment settings object at that point (it's all OS widgets).  

So, we should say *something* ... but we need to figure out what... but it's not a "clients" thing.

**Comment by marcoscaceres** [comment]
Also the conclusion is that icons are not critical sub-resources, because the UA can (and often does) provide a fallback icon.

**Comment by dmurph** [comment]
I think this pull request closes this if it is merged:
https://github.com/w3c/manifest/pull/1171

I would also say they are not critical - we don't need the images to be fetched / non-200'd to just parse the manifest. The UA may make decisions based on whether they can be fetched or not, but I'm very happy saying that is up to the UA.
